osu.Game/Rulesets/UI/DrawableRuleset.cs

```diff
@@ -38,9 +41,8 @@ namespace osu.Game.Rulesets.UI
     public abstract class DrawableRuleset<TObject> : DrawableRuleset, IProvideCursor, ICanAttachKeyCounter
         where TObject : HitObject
     {
-        public override event Action<JudgementResult> OnNewResult;
-
-        public override event Action<JudgementResult> OnRevertResult;
+        public override event Action<JudgementResult> NewResult;
+        public override event Action<JudgementResult> RevertResult;
```

Comment:
I can revert this if you'd like, it is a bit of a breaking change but is unlikely to be used by any rulesets.

The reason for the rename is to match `Playfield` + `DrawableHitObject` + `HitObjectContainer`.